docs/src/environments/_index.md

```diff
@@ -54,106 +89,91 @@ Each new environment you create is considered a **child** of the **parent** envi
 
-- **Agile**: A child environment per sprint.
-  Each story in the sprint can have its own environment as a child of the sprint environment.
-- **Developer-centric**: One QA environment and a few development environments
-  (*per developer*, *per task*, or similar).
-- **Testing**: An operational test environment, a user test environment, and a few unit test environments.
-- **Hotfix**: One environment for every bug, security, or hotfix that needs deployment.
+-   **Agile**: A child environment per sprint.
+    Each story in the sprint can have its own environment as a child of the sprint environment.
 
-Here is an example of a possible Agile workflow.
+-   **Developer-centric**: One QA environment and a few development environments
+    (*per developer*, *per task*, or similar).
 
-The administrator started by [branching](../other/glossary.md#branch) the Live environment to create a Sprint environment.
-They then give each developer permission to branch the Sprint environment to create new feature environments.
+-   **Testing**: An operational test environment, a user test environment, and a few unit test environments.
 
-![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
+-   **Hotfix**: One environment for every bug, security, or hotfix that needs deployment.
 
-As Feature 1 is developed, work is reviewed by accessing the deployed Feature 1 environment.
-When the review is done, Feature 1 is [merged](../other/glossary.md#merge) into the Sprint environment.
+#### Example workflow
 
-Once the Sprint environment has Feature 1,
-the remaining features [sync](../other/glossary.md#sync) with the Sprint environment.
-This ensures their working environment is up-to-date with the latest code.
+Example of an Agile workflow :
 
-![Features from the Sprint environment are synced with the Feature 2 environment](/images/workflow/sync.svg "0.2")
+1.  The admin [branches](../other/glossary.md#branch) the Live environment to create a Sprint environment.
 
-When the sprint is complete and all features merged into the Sprint environment,
-the administrator can make a backup of the live site.
-Then they can merge the Sprint environment into the Live environment.
+2.  The admin gives each developer permission to branch the Sprint environment to create new feature environments.
 
-![Features from the Sprint environment are merged into the Live environment](/images/workflow/merge-live.svg "0.2")
+    ![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
 
-The administrator can then sync the next sprint's environment with the Live environment
-to repeat and continue the development process.
+3.  Feature 1 is developed and work is reviewed by accessing the deployed Feature 1 environment.
```

Comment:
:lady_beetle: It seems like an image of this step would be nice, given that all the others have it.